datumaro/util/meta_file_util.py

```diff
@@ -109,6 +109,9 @@ def save_hashkey_file(path, item_list):
 
     if osp.isdir(path):
         meta_file = get_hashkey_file(path)
+    hashkey_folder_path = osp.join(path, "hash_key_meta")
```

Comment:
Please create a constant variable for "hash_key_meta" also such as `DATASET_HASHKEY_FILE`.